Adjustments needed to get memcached version 1.4.35 to build with the Studio
compiler on Solaris.

These changes will be passed upstream.

--- memcached-1.4.35/logger.c.orig	2017-03-01 15:11:00.811829274 +0000
+++ memcached-1.4.35/logger.c	2017-03-01 15:11:09.009200863 +0000
@@ -11,6 +11,10 @@
 #include <atomic.h>
 #endif
 
+#if !defined(HAVE_GCC_64ATOMICS) && defined(__sun)
+#include <atomic.h>
+#endif
+
 #include "memcached.h"
 #include "bipbuffer.h"
 
--- memcached-1.4.35/memcached.c.orig	2017-03-01 15:10:36.902614850 +0000
+++ memcached-1.4.35/memcached.c	2017-03-01 15:50:09.188544377 +0000
@@ -364,8 +364,6 @@
                     sleep_time);
         usleep((useconds_t) sleep_time * 1000000);
     }
-
-    return NULL;
 }
 
 static int start_conn_timeout_thread() {
